doc/HardFloat-Verilog.html

```diff
@@ -1501,12 +1501,14 @@ <H3>9.7. Fused Multiply-Add (<CODE>mulAddRecFN</CODE>,
     mulAddRecFN#(parameter expWidth, parameter sigWidth) (
         input [(`floatControlWidth - 1):0] control,
         input [1:0] op,
+        input int_mul,

```

Comment:
In general we prefer that for control signals, we avoid ambiguity of interface where you have parallel input control signals indicating different transactions but the module cannot actually do those simultaneous operations. Does it make sense instead to merge this into a single 3-bit op?

Reply:
Since we're forking hardfloat to add this, it may make sense to make a pkg which has a enum for this opcode, and sig/exp/rec widths for ieee single / double precision.

Reply:
@dpetrisko Hi Dan, the opcode has its original meaning. One the other hand. I believe standard Verilog does not support enumeration and pkg unless we'll update to SV, which means test benches for iverilog will be relinquished.

Reply:
Hm interesting.  I believe Icarus supports enums.  We could make an include file instead of a package to get around a lack of package support.  Usually, we are amenable to enums and packages because VCS and Verilator both support them

Reply:
Let's see if iverilog will support the package.